### ISSUE TITLE
Unturned scripts

### ISSUE THREAD

**Issue by JorelDecker** [SmartlyDressedGames/Unturned-3.x-Community#2579]
I added a script to the grenade, after which I received fire damage on the server. But here I needed the fire effect to work when hitting the ground. Then I added a fire script to the effect that is called after the" explosion " of the grenade, but after that the effect stopped working on the server. That is, if you attach it to a grenade, everything works, and if it is a separate effect that is caused by a grenade, then it does not work on the server (Although in a single game everything is fine). Please help me synchronize this with the server
![image](https://user-images.githubusercontent.com/83378019/116483922-ba57b800-a890-11eb-8a5a-9413a6599651.png)

**Comment by JorelDecker** [comment]
![image](https://user-images.githubusercontent.com/83378019/116483965-d0657880-a890-11eb-8cf2-29d2f6a5fe3b.png)

**Comment by SDGNelson** [comment]
Are you using something which spawns it as an EffectAsset? In that case you can set `Splatter_Temperature Burning` in the effect .dat file, and it will respect the PvP settings properly.

**Comment by JorelDecker** [comment]
> Are you using something which spawns it as an EffectAsset? In that case you can set `Splatter_Temperature Burning` in the effect .dat file, and it will respect the PvP settings properly.

Yes, thank you, I have already found a solution, now the same problem for the barricade on the radioactive zone, works in solitary, but not on the server. Removes radiation, but the damage does not go.
![image](https://user-images.githubusercontent.com/83378019/116632746-8c41a900-a960-11eb-9266-c140376ea350.png)
![image](https://user-images.githubusercontent.com/83378019/116632772-96fc3e00-a960-11eb-8f37-e116502ffb95.png)

**Comment by SDGNelson** [comment]
Sorry, but there is no intended way for dynamically spawning deadzones. Some mods have found workarounds but it is not supported yet.
